checkbox/app.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright 2017-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Messenger Platform Quick Start Tutorial
+ *
+ * This is the completed code for the Messenger Platform quick start tutorial
+ *
+ * https://developers.facebook.com/docs/messenger-platform/getting-started/quick-start/
+ *
+ * To run this code, you must do the following:
+ *
+ * 1. Deploy this code to a server running Node.js
+ * 2. Run `npm install`
+ * 3. Update the VERIFY_TOKEN
+ * 4. Add your PAGE_ACCESS_TOKEN to your environment vars
+ *
+ */
+
+const PAGE_ACCESS_TOKEN = "";
+
+// Imports dependencies and set up http server
+import request from 'request'; // creates express http server
+
+import express from 'express';
+import body_parser from 'body-parser';
+import path from 'path';
```

Comment:
@ChrisChinchilla Is this dependency needed? I'm not too familiar but it looks like your usage below of:

path.join(__dirname, '/public', 'thanks.html')

is the same as __dirname+'/public'+'/thanks.html';

If possible, let's avoid including any unnecessary dependencies.